spring-boot-project/spring-boot/src/main/resources/org/springframework/boot/logging/logback/defaults.xml

```diff
@@ -6,15 +6,16 @@ Default logback configuration provided for import
 
 <included>
 	<conversionRule conversionWord="applicationName" converterClass="org.springframework.boot.logging.logback.ApplicationNameConverter" />
+	<conversionRule conversionWord="applicationGroup" converterClass="org.springframework.boot.logging.logback.ApplicationGroupConverter" />
```

Comment:
Minor nit: these rules are ordered based the conversion word so `applicationGroup` should go before `applicationName`.